tests/acceptance/fixtures.py

```diff
@@ -336,6 +336,11 @@ def prepared_test_build_base(request, bitbake_variables):
         build_dir = tempfile.mkdtemp(prefix="test-build-", dir=os.environ['BUILDDIR'])
 
     local_conf = os.path.join(build_dir, "conf", "local.conf")
+
+    # All acceptance tests must handle a read-only filesystem.
+    add_to_local_conf({'local_conf': os.path.join(build_dir, "conf", "local.conf")},
```

Comment:
@kacf will this be the appropriate way of adding the read-only-rootfs functionality to all the acceptance tests?